contracts/base/LoansBase.sol

```diff
@@ -568,18 +564,50 @@ contract LoansBase is LoansInterface, Base {
         internal
         view
         returns (uint256)
+    {
+        uint256 loanAmount = _getTotalOwed(loanID);
+        uint256 collateralRatio = loans[loanID].loanTerms.collateralRatio;
+        return loanAmount.mul(collateralRatio).div(TEN_THOUSAND);
+    }
+
+    /**
+        @notice Returns the total amount owed for a specified loan
+        @param loanID The id of the loan to get the total amount owed
+     */
+     function _getTotalOwed(uint256 loanID)
+        internal
+        view
+        returns (uint256)
     {
         TellerCommon.LoanStatus currentStatus = loans[loanID].status;
         uint256 loanAmount;
         if (currentStatus == TellerCommon.LoanStatus.TermsSet) {
-            loanAmount = loans[loanID].loanTerms.maxLoanAmount;
+            uint256 interestOwed = _getInterestOwed(loanID, loans[loanID].loanTerms.maxLoanAmount);
+            loanAmount = loans[loanID].loanTerms.maxLoanAmount.add(interestOwed);
         } else if (currentStatus == TellerCommon.LoanStatus.Active) {
-            loanAmount = getTotalOwed(loanID);
+            loanAmount = loans[loanID].interestOwed.add(loans[loanID].principalOwed);
         } else {
-            return 0;
+            loanAmount = 0;
```

Comment:
also here you can just return 0. no need to set the variable

Reply:
Done